operator/hack/lokistack_dev.yaml

```diff
@@ -9,3 +9,7 @@ spec:
     secret:
       name: test
   storageClassName: standard
+  template:
```

Comment:
I propose not to change this file ... instead, create a new one called `lokistack_ruler_dev.yaml` that will deploy the ruler component  (and will have the changes from here)

We need to somehow show that the deployment of the ruler is configurable.